src/empire/game.py

```diff
@@ -1,70 +1,42 @@
-import dataclasses
+import logging
 
 import pygame
 
+import service_locator
```

Comment:
is it used?

Reply:
yes
        self.config = service_locator.get_game_config()